azure-pipelines.yml

```diff
@@ -0,0 +1,32 @@
+trigger:
+  branches:
+    include: ['*']
+  tags:
+    include: ['*']
+
+jobs:
+  - job: Test
+  pool:
+    vmImage: 'Ubuntu-18.04'
+    vmImage: 'macOS-10.13'
+    vmImage: 'vs2017-win2016'
+  strategy:
+    matrix:
+      Python36:
+        python.version: '2.7'
+      Python37:
+        python.version: '3.6'
+    maxParallel: 2
```

Comment:
why limit the parallelism? Faster is better!

Reply:
Ok, i will remove this.